janusgraph-backend-testutils/src/main/java/org/janusgraph/graphdb/JanusGraphTest.java

```diff
@@ -4315,6 +4323,18 @@ public void testTinkerPopOptimizationStrategies() {
         assertTrue(queryProfilerAnnotationIsPresent(t, QueryProfiler.MULTIQUERY_ANNOTATION));
     }
 
+    private static void assertSameResultWithOptimizations(Traversal<?,?> originalTraversal, TraversalStrategy<?>... strategies) {
```

Comment:
Looks good. Do you want to move this to JanusGraphAssert?

Reply:
I did this quick and dirty because I have an ongoing PR (#2202) to reorganize optimization strategies and the according tests. I will simply refactor the tests for this Fix together with (#2202).